postman/threads-api.postman_collection.json

```diff
@@ -1,11 +1,11 @@
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
-		"_exporter_id": "29882223",
-		"_collection_link": "https://www.postman.com/meta/threads/collection/dht3nzz/threads-api?action=share&source=collection_link&creator=29882223"
+		"_exporter_id": "34767301",
+		"_collection_link": "https://threadsapi.postman.co/workspace/Threads~5610276e-a2dd-4bd2-bd79-301d93b9c96a/collection/34473965-a93be385-2264-4766-826f-b53cb6f50d7e?action=share&source=collection_link&creator=34767301"
```

Comment:
Should the collection link be pointing to https://threadsapi.postman.co or https://www.postman.com/meta/threads?

Reply:
Thanks for catching this! Fixed